docs/core/development/policy/coding-standard.rst

```diff
@@ -229,6 +230,8 @@ Docstrings should always be used to describe the purpose of methods, functions,
 Moreover, all methods, functions, classes, and modules must have docstrings.
 In addition to documenting the purpose of the object, the docstring must document all of parameters or attributes of the object.
 
+Docstrings must be reflowed at 79 characters.
```

Comment:
If someone could turn https://github.com/glyph/python-docstring-mode/blob/master/docstring_wrap.py into a tool that ran over a codebase's docstrings rather than just an editor plugin shim helper gizmo, we could possibly Black-ify our epytext as well...

Reply:
This docstring wrapper makes a number of assumptions about the format of the docstrings it's wrapping, most of which are going to be wrong for a lot of projects.  I don't think there's any way to realistically make it universal enough for Black to consider incorporating it.

Reply:
Are the assumptions specific to Twisted or specific to particular docstring formats? In the latter case, being able to specify the docstring format to Black and/or supporting the `__docformat__` constant could be used to turn the docstring reformatting on/off.